resources/assets/js/bootstrap.js

```diff
@@ -5,41 +5,17 @@
  */
 
 try {
-    window.$ = window.jQuery = require('jquery');
+  window.$ = window.jQuery = require("jquery");
 } catch (e) {}
 
-/**
- * We'll load the axios HTTP library which allows us to easily issue requests
- * to our Laravel back-end. This library automatically handles sending the
- * CSRF token as a header based on the value of the "XSRF" token cookie.
- */
-
-window.axios = require('axios');
-
-window.axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
-
-/**
- * Next we will register the CSRF Token as a common header with Axios so that
- * all outgoing HTTP requests automatically have it attached. This is just
- * a simple convenience so we don't have to attach every token manually.
- */
-
-let token = document.head.querySelector('meta[name="csrf-token"]');
-
-if (token) {
-    window.axios.defaults.headers.common['X-CSRF-TOKEN'] = token.content;
-} else {
-    console.error('CSRF token not found: https://laravel.com/docs/csrf#csrf-x-csrf-token');
-}
-
-import Echo from 'laravel-echo'
+import Echo from "laravel-echo";
 
-window.Pusher = require('pusher-js');
+window.Pusher = require("pusher-js");
 
 window.Echo = new Echo({
-    broadcaster: 'pusher',
-    key: window.PusherAppKey,
-    cluster: 'us2',
-    encrypted: true,
-    namespace: 'Chompy.Events'
+  broadcaster: "pusher",
+  key: window.PusherAppKey,
+  cluster: "us2",
+  encrypted: true,
+  namespace: "Chompy.Events"
```

Comment:
Not a big deal at all should these double quotations be single?

Reply:
Oh awk, I think my Prettier extension got a little too excited.

Reply:
Configured Prettier for this app & fixed those quotes up in 1547fcb.